manim/mobject/svg/tex_mobject.py

```diff
@@ -621,6 +621,21 @@ def fade_all_but(self, index_or_string, opacity=0.5):
 
 
 class Title(Tex):
+    """
+    Example
+    --------
+    .. manim:: TitleExample
+        :save_last_frame:
+
+        import manim
+        class TitleExample(Scene):
+            def construct(self):
+                banner = ManimBanner()
```

Comment:
Can you make it light theme and keep the version down?

Reply:
Title has no option to be at the buttom, see here:
```
class Title(Tex):
    def __init__(
        self,
        *text_parts,
        scale_factor=1,
        include_underline=True,
        match_underline_width_to_text=False,
        underline_buff=MED_SMALL_BUFF,
        **kwargs,
    ):
   ```
and as the default color is white for the text, I want to keep it dark theme